build-apisix-base.sh

```diff
@@ -20,6 +21,7 @@ else
     wasm_nginx_module_ver="0.6.4"
     lua_var_nginx_module_ver="v0.5.3"
     grpc_client_nginx_module_ver="v0.3.1"
+    amesh_ver="main"
```

Comment:
Amesh hasn't yet released a version with `make install`, so here is the main branch for now.